package_managers/installer.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Some comments at the top explaining what this is for, and what files it expects would be helpful in the future.